### PR TITLE
Implement UI elements for usage stats opt-out/opt-in

### PR DESCRIPTION
**Pull request by bernd**
Depends on Graylog2/graylog2-server#1528.

### CODE REVIEW

**Comment by joschi** [comment]
What will happen if the Anonymous Usage Statistics plugin is either deactivated or not installed at all?

**Comment by bernd** [comment]
> What will happen if the Anonymous Usage Statistics plugin is either deactivated or not installed at all?

I am currently testing that. :smile:

File: javascript/src/components/usagestats/UsageStatsOptIn.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    <Row className="content">
+                        <Col md={12}>
+                            <h2><i className="fa fa-bar-chart"></i> Anonymous Usage Statistics</h2>
+                            {form}
+                        </Col>
+                    </Row>
+                );
+            }
+        } else {
+            UsageStatsOptOutStore.getOptOutState().done((optOutState) => {
```
Comment (edmundoa) [inline javascript/src/components/usagestats/UsageStatsOptIn.jsx]: I would move this into [`componentDidMount`](https://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount), in that way there's a more clear separation of loading data and rendering.

Comment (bernd) [inline javascript/src/components/usagestats/UsageStatsOptIn.jsx]: Thanks, done!

**Comment by joschi** [comment]
The opt-in/opt-out setting on the System/Overview page are currently only shown if the user decided on whether to opt-out or stay opted-in on the getting started page. This is a bit irritating and users who just closed the getting started page won't ever see those settings. The opt-in/opt-out setting on the System/Overview page should always be shown, if the plugin was loaded and is enabled.

**Comment by joschi** [comment]
LGTM. :+1:

**Pull request merged**